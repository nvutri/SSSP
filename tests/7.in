c random graph n=32 m=128 lo=0 hi=16384 seed=17630404
p sp 32 128
a 10 16 13030
a 16 21 15343
a 21 14 5961
a 14 13 6080
a 13 26 3360
a 26 27 6208
a 27 8 5613
a 8 18 2883
a 18 9 2352
a 9 15 1068
a 15 3 10168
a 3 24 1487
a 24 28 1083
a 28 32 6384
a 32 4 5770
a 4 6 14491
a 6 20 14186
a 20 29 4616
a 29 17 14575
a 17 1 8949
a 1 25 10126
a 25 22 9920
a 22 7 9992
a 7 23 3834
a 23 30 4042
a 30 31 585
a 31 11 6880
a 11 5 3664
a 5 2 10656
a 2 19 4340
a 19 12 8748
a 12 10 7064
a 9 25 7429
a 3 27 9148
a 2 4 5968
a 19 20 4754
a 27 11 3272
a 13 24 13131
a 6 5 9153
a 8 9 3812
a 30 4 7763
a 23 30 712
a 19 8 14587
a 25 22 1584
a 3 28 2445
a 30 5 3610
a 2 9 13803
a 7 29 212
a 28 23 122
a 12 4 6785
a 9 25 8080
a 17 6 8264
a 1 13 11536
a 13 25 15355
a 9 15 16091
a 5 18 2339
a 29 14 1959
a 32 11 14880
a 16 11 6033
a 17 28 1783
a 17 13 12046
a 24 5 1261
a 27 7 11993
a 29 17 15379
a 21 2 5010
a 6 9 1005
a 1 31 12416
a 25 13 5085
a 9 20 3758
a 9 26 12661
a 9 4 13535
a 23 26 8939
a 9 29 374
a 28 18 3981
a 10 14 11928
a 9 27 553
a 5 20 9855
a 24 8 2195
a 7 25 9732
a 19 26 13332
a 31 16 3111
a 22 2 7055
a 25 15 13364
a 16 29 4613
a 19 16 803
a 23 2 9035
a 7 18 9627
a 1 10 11489
a 29 17 16168
a 17 11 9
a 3 32 10588
a 11 20 10566
a 1 19 16368
a 15 22 5279
a 20 3 4676
a 22 24 7900
a 27 10 16240
a 9 6 13250
a 5 29 7483
a 2 32 7569
a 22 30 11415
a 28 2 2839
a 20 8 3330
a 19 3 7245
a 11 13 2369
a 15 3 8512
a 25 1 3370
a 15 4 15188
a 2 17 10320
a 3 15 5835
a 4 24 16357
a 31 17 11078
a 19 21 6352
a 11 32 3420
a 29 11 13870
a 26 22 3042
a 19 17 14645
a 11 5 2115
a 23 29 15919
a 25 4 477
a 13 20 14324
a 32 17 10505
a 28 9 10505
a 7 13 4724
a 18 32 12362
a 2 18 3226
a 29 25 11039
a 19 14 1737
