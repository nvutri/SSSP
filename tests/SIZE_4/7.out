1 0
2 18765
3 23613
4 10603
5 20871
6 25094
7 21702
8 21805
9 24343
10 11489
11 19296
12 25116
13 11536
14 18105
15 23490
16 15527
17 23494
18 21991
19 16368
20 21122
21 22720
22 11710
23 20815
24 19610
25 10126
26 14896
27 21104
28 20693
29 20140
30 21527
31 12416
32 22716
